convert-hf-to-gguf.py

```diff
@@ -1195,6 +1208,7 @@ def set_vocab(self):
             self._set_vocab_qwen()
 
     def set_gguf_parameters(self):
+        super().set_gguf_parameters()
```

Comment:
This is unusual. Doesn't this duplicate some KV fields in the model?

Reply:
Removed